CSS/main.css

```diff
@@ -0,0 +1,200 @@
+@import url(http://fonts.googleapis.com/css?family=Yellowtail);
+@import url(http://fonts.googleapis.com/css?family=Roboto:900);
+@import url(http://fonts.googleapis.com/css?family=Oswald:700);
+@import url(http://fonts.googleapis.com/css?family=Open+Sans+Condensed:300);
+@import url(http://fonts.googleapis.com/css?family=Scheherazade);
+
+html, body,
+h1, h2, h3, h5, p, a, code, img, samp,
+small, strike, strong, center, ul, li,
+article, aside, footer, header,
+menu, nav, section, summary {
+  margin: 0;
+  padding: 0;
+  border: 0;
+  font-size: 100%;
+  font: inherit;
+  vertical-align: baseline;
+}
+
+/* HTML5 display-role reset for older browsers */
+
+article, aside, footer, header, menu, nav, section {
+  display: block;
+}
+body {
+  line-height: 1;
+}
+ul {
+  list-style: none;
+}
+blockquote, q {
+  quotes: none;
+}
+blockquote:before, blockquote:after,
+q:before, q:after {
+  content: '';
+  content: none;
+}
+
+/* set border margin*/
+
+body{
+  margin: 2em 6em;
+  width:900px;
+}
+
+/*logo styling*/
+
+.logoStyleOne {
```

Comment:
Make the indentation of this rule match the rest of the document. Try to keep consistent indentation.